sklearn/feature_extraction/text.py

```diff
@@ -971,32 +971,50 @@ def _make_int_array():
 class TfidfTransformer(BaseEstimator, TransformerMixin):
     """Transform a count matrix to a normalized tf or tf-idf representation
 
-    Tf means term-frequency while tf-idf means term-frequency times inverse
-    document-frequency. This is a common term weighting scheme in information
-    retrieval, that has also found good use in document classification.
+    Definitions:
 
-    The goal of using tf-idf instead of the raw frequencies of occurrence of a
-    token in a given document is to scale down the impact of tokens that occur
-    very frequently in a given corpus and that are hence empirically less
-    informative than features that occur in a small fraction of the training
-    corpus.
+    Term-frequency (tf) = the frequency of a term in a document.
+
+    Inverse-document-frequency (idf) = 1 / (# of docs a term appears in)
+
+    This is a common term weighting scheme in information
```

Comment:
Maybe "family of schemes". Really you mean that the product of a monotonic increasing function of TF and a monotonic increasing function of IDF constitutes a family of term weighting schemes. You miss the fact that we care about their product.